LIPS/lip-4.md

```diff
@@ -0,0 +1,47 @@
+---
+lip: 4
+title: 72h Aragon Votes
+status: WIP
+author: Victor Suzdalev (kadmil)
+discussions-to: https://research.lido.fi/t/increase-the-dao-voting-duration/1048
+created: 2021-09-28
+updated: -
+---
+
+# 72h Aragon Votes
+
+Currently Aragon votes on Lido DAO are 24h long. This timeframe has proved to be short enough for reaching the quorum even on non-contentios votes. While most routine DAO operations are expected to switch to EasyTracks (vehicle for pre-approved opt-out DAO actions), any custom blockchain action on behalf of DAO would still require the Aragon vote. The proposal is to make Aragon votes longer, allowing more LDO holders to participate.
+
+## Problem
+
+Gathering the quorum on Aragon votes within 24h window proved difficult enough even for repetitive and non-contentios actions such as raising Node Operators' keys limits and refilling liquidity pools' rewards programs.
+
+## Proposed solution
+
+The proposed solution is to make Aragon votes last 72h instead of the current 24h. To achieve this, the DAO would have to upgrade the Voting smart contract, as the current version doesn't allow changing the vote duration.
+
+The Voting contract version where the vote duration can be changed can be found [there](https://github.com/lidofinance/aragon-apps/blob/8c46da8704d0011c42ece2896dbf4aeee069b84a/apps/voting/contracts/Voting.sol)
+
+We've strived to make as little changes as possible so not to introduce any issues to the Voting contract the DAO currently uses.
+
+## Change implementation
+
+The upgrade plan is:
+1. Deploy the Voting implementation from [https://github.com/lidofinance/aragon-apps/blob/8c46da8704d0011c42ece2896dbf4aeee069b84a/apps/voting/contracts/Voting.sol](https://github.com/lidofinance/aragon-apps/blob/8c46da8704d0011c42ece2896dbf4aeee069b84a/apps/voting/contracts/Voting.sol).
+2. Run the Aragon vote with actions:
+   1. Push new app version to the Aragon repo
```

Comment:
a bit more color here - you mean aragon pm? what's aragon repo?

Reply:
Marked that it's our own fork of Aragon repo